submissions/Olumide Jokotade/HTML-CSS/Frere Wears Homepage.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
```

Comment:
Indentation is not too good here. Preferably move these CSS to a separate file and import it here